testdata/baselines/reference/submoduleAccepted/conformance/typeTagModuleExports.types.diff

```diff
@@ -1,11 +1,13 @@
 --- old.typeTagModuleExports.types
 +++ new.typeTagModuleExports.types
-@@= skipped -4, +4 lines =@@
+@@= skipped -2, +2 lines =@@
+ === bug27327.js ===
+ /** @type {string} */
  module.exports = 0;
- >module.exports = 0 : string
+->module.exports = 0 : string
++>module.exports = 0 : 0
```

Comment:
I think this is an improvement, in which the binary expression source of the export ignores the annotation, but usages of it have the annotated type.